### ISSUE TITLE
isues with puppeteer / sharp 

### ISSUE THREAD

**Issue by acognigni-evotecnia** [wppconnect-team/wppconnect#2038]
## Description

i woke up today to fin that my project had crashed with the following error

    Error: Execution context was destroyed, most likely because of a navigation.

i saw there was an update for wpconnect so i installed it and now sharp was refusing to work with the following error
 
    Could not load the "sharp" module using the win32-x64 runtime

i updated node to 21.6.0 and now in my dev pc i still get the same error, while on the server it hangs at 

````
info:
  _       ______  ____  ______                            __
  | |     / / __ \/ __ \/ ____/___  ____  ____  ___  _____/ /_
  | | /| / / /_/ / /_/ / /   / __ \/ __ \/ __ \/ _ \/ ___/ __/
  | |/ |/ / ____/ ____/ /___/ /_/ / / / / / / /  __/ /__/ /_
  |__/|__/_/   /_/    \____/\____/_/ /_/_/ /_/\___/\___/\__/
info:     Checking for updates
(node:680144) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative inste
ad.
(Use `node --trace-deprecation ...` to show where the warning was created)
info:     You're up to date
info:     [session:browser] Using browser folder 'C:\backend\<projectName>\tokens\session'
info:     [session:browser] Initializing browser...

  Puppeteer old Headless deprecation warning:
    In the near future `headless: true` will default to the new Headless mode
    for Chrome instead of the old Headless implementation. For more
    information, please see https://developer.chrome.com/articles/new-headless/.
    Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`
    If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.
````

## Environment

* **WPPConnect version(s):** ^1.23.0
* **WA-JS version(s):** 2.28.1
* **Browser:** Chrome 109.0.5414.168 (Chrome doesn't support my Server's OS so it's not up to date)
* **OS:** Windows windows Server 2012 R2 Standard
* **Node version:** Node  started with: 14.17.6 ended with : 21.6.0
* **WhatsApp version:** 2.23.25.83
* **MultiDevice (BETA):** no

## Additional context / Screenshot

since i have a lot of projects running with node on that server, whenever a project requires a different version i set it up with `npm i node@21.6.0` it shouldn't cause any errors, but it might.

**Comment by acognigni-evotecnia** [comment]
what is the reccomended node verision for this project? and os?

it is likely that it could have worked without changing the node version if i had a linux server, so i wanted to ask if  windows is fully supported or you only test in linux

**Comment by Flamerax** [comment]
I have a same problem in my code, shows this error:

`Error: Could not load the "sharp" module using the win32-x64 runtime`

i use node version 20.9.0 and @wppconnect-team/wppconnect@1.28.4 in Windows x64

Any solution?

**Comment by acognigni-evotecnia** [comment]
i knew i was forgetting to say something important:

my coworker was getting the same errors, then followed the same steps and we got it working on their workstation
even with this we haven't been able to fix the issue in the server

Also, hopefully irrelevant: since today at 14:30 UTC i got a few windows security warnings in the event viewer. like 3 every 30 minutes until 16:30 UTC

**Comment by dudanogueira** [comment]
I have seen some sharp issues in the past with old linux boxes.

What are you running?

the fact that it worked in a different linux box indicates the issue may be in the box.

this seems related:
https://github.com/lovell/sharp/issues/3926

**Comment by acognigni-evotecnia** [comment]
> I have seen some sharp issues in the past with old linux boxes.
> 
> What are you running?
> 
> the fact that it worked in a different linux box indicates the issue may be in the box.
> 
> this seems related: [lovell/sharp#3926](https://github.com/lovell/sharp/issues/3926)

sorry if i didn't explain it properly . it's all windows: my pc, my coworker's and the server

**Comment by acognigni-evotecnia** [comment]
Another important piece of information: Chrome stopped supporting my server's OS so it's no longer up to date ( it detects it as windows 8.1 i think)

**Comment by dudanogueira** [comment]
Oh, sorry. didn't read at the issue. Windows is most likely the root cause ;)

**Comment by dcrec1** [comment]
Im having this issue on Linux.

**Comment by renat473** [comment]
FIX Version NodeJS 18.17.0

```sh
npm install --force @img/sharp-linux-x64
yarn add sharp
yarn add sharp --ignore-engines
```
